Source/App/EbAppProcessCmd.c

```diff
@@ -13,7 +13,7 @@
 #include "EbAppContext.h"
 #include "EbAppConfig.h"
 #include "EbSvtVp9ErrorCodes.h"
-#include "EbSvtVp9Time.h"
+#include "EbAppTime.h"
 
```

Comment:
I think EbTime.h is better, instead of EbAppTime.h. First, to keep consistent with av1, second, easier for developer to remember the file name and search.

Reply:
Problem with EbTime.h is that it's already used as a name in Source/Lib/Codec, so using EbTime.h will only cause more confusion

Reply:
Right now it's
```none
./Source/Lib/Codec/EbTime.h
./Source/Lib/Codec/EbTime.c
./Source/App/EbAppTime.h
./Source/App/EbAppTime.c
```

Reply:
AV1 has the same file name since the module has the same functionality. But it is not a bit deal. Either is fine.